timed/reports/management/commands/notify_reviewers_unverified.py

```diff
@@ -95,13 +112,25 @@ def _notify_reviewers(self, start, end, reports):
                         # we need start and end date in system format
                         'start': str(start),
                         'end': str(end),
+                        'message': message,
                         'reviewer': reviewer,
                         'protocol': settings.HOST_PROTOCOL,
                         'domain': settings.HOST_DOMAIN,
                     }, using='text'
                 )
 
-                mails.append((subject, body, from_email, [reviewer.email]))
+                message = EmailMessage(
+                    subject=subject,
+                    body=body,
+                    from_email=from_email,
+                    to=[reviewer.email],
+                    connection=connection
+                )
+
+                if cc:
```

Comment:
this `if` is not necessary. Just make no default for cc as EmailMessage expects None to cc for not configured.

Reply:
Done.